draft-ietf-quic-multipath.md

```diff
@@ -132,18 +132,24 @@ and the actual decision to setup or tear down paths are assumed
 to be handled by the application. But this document does not prevent future extensions from
 defining mechanisms to cope with the remaining scenarios.
 
-Further, this document does not specify detailed algorithms that define
+Further, this document does not specify scheduling algorithms that define
 how multiple, simultaneously open paths are used to send packets.
-As these differ depending on the application requirements,
+As these differ depending on application requirements,
 only some basic implementation guidance is discussed in {{impl-consideration}}.
+This extension can be used with different scheduling algorithms that,
+e.g., can range from support for failover to simulatenous
+use of the aggreate capacity across all active paths.
+There are IETF specifications for path fail-over, but
+there are currently no IETF standards-track specifications for simultaneously
+(concurrently) using the aggregated capacity of a set of active paths.
```

Comment:
Do we need to say „aggregated capacity“ here again?

This would be less wordy:
```suggestion
(concurrently) using multiple paths.
```

Reply:
+1

Reply:
Sure, I could live without "aggregated capacity" here if the word concurrently is understood to say that.